### ISSUE TITLE
Removing Show from library should remove show/episodes from failed downloads queue 

### ISSUE THREAD

**Issue by Silentbob101** [SickRage/sickrage-issues#1011]
When you remove a show from your library database, and select the actual files to be deleted this action should remove any episodes that are on the failed downloads list. Otherwise Sickrage will still be looking for them and when it finds them, they are downloaded and reads the show and episode to your library again. (i have been stung by this a couple of times)

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by neoatomic** [comment]
Not sure if i can follow @Silentbob101 , So if you remove an episode from your hard disk the episode will get set to skipped in Sickrage. 

Could you explain a little more of what you mean.? And also include your version info/logs and examples to replicate.?

**Comment by Silentbob101** [comment]
Ok... 

i have 'iZombie' tv show in my Library but i am missing S1E1. SickRage finds a nzb for S1E1.nzb trys to download it... Download fails.  S1E1.nzb is added to my failed downloads list.

I decide i dont was this show in my library any more i click on the show and click remove, i select the option to delete the files as well.. Yay its all gone. But i S1E1.nzb is still in my failed downloads queue, so Sick rage still searches for a different nzb S1E1 of iZombie even though the show is no longer In my database. when it finds an alternate S1E1.nzb it starts download and if successful re adds the show to my library.

In short, on deletion of a show from the library database. All failed downloads for episodes of that show should also be removed from the failed downloads queue.

i will post the logs/version number when i get home later today... hope it makes sense!

**Comment by neoatomic** [comment]
Thanks for the clarification. That indeed shouldn't happen.
On what version are you on.?

**Comment by OmgImAlexis** [comment]
@neoatomic I don't think they should be removed from failed downloads though maybe just add a check to only research failed downloads if the show is still added. That way people can add the show back if they'd like and the history of the failed downloads is still there.
